change/@azure-msal-browser-14b0229b-98b8-445e-b79a-e49d634452d1.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "patch",
```

No API change, can be patch.